src/components/RoomView/RoomView.tsx

```diff
@@ -284,65 +288,62 @@ const RoomView = (props: ClassName) => {
   const claimHost = () => {
     update(hostRef, { playerId, username });
     update(child(playersInRoomRef, playerId), { playerId, username, isHost: true });
-  };
-
-  return (
-    <div className="roomViewBG">
-      <div className={props.className}>
-        <WelcomeBoard />
-
-        <div className="gameStatusClaimHost">
-          {!host && (
-            <p>
-              The host has left, <button onClick={claimHost}>claim host responsibilities</button> to begin game.
-            </p>
-          )}
-          <div className="gameStatus">
-            <GameStatus />
-          </div>
-        </div>
-
-        {/* is there isnt at least one person to each role, setup board should be disabled / not visible 
-      is host AND there is at least one person on each team */}
-        {isHost && (
-          <Popup
-            trigger={
-              <Button style={{ display: 'block', marginLeft: 'auto', marginRight: 'auto' }}> Set Up Board </Button>
-            }
-          >
-            <SetupGame />
-          </Popup>
-        )}
-
-        <div className="flexBox">
-          <TeamOneBox />
-          <div className="boardContainer">
-            {/* player is operative && show operative board, otherwise theyre a spymaster*/}
-            {/* this is working for now, but we probably need more protection to not display 
+  }
+  return ( <div className="roomViewBG">
+    <div className={props.className}>
+      <WelcomeBoard />
+      <div className="gameStatusClaimHost">
+      <GameStatus />
```

Comment:
because the outside div contains two functions: gameStatus and ClaimHost, therefore the outside div name is gameStatusClaimHost, the innder div is gameStatus, I am using gameStatus on claimHost function as well for now but open the possibility of styleing it separtely, in that case, will change the className to claimHost

Reply:
ok! changing the names sounds good, i just wanted to make sure of the intended order!